modules/cnv-about-dedicated-resources.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-about-dedicated-resources_{context}"]
+
+= About dedicated resources
+
+When you enable dedicated resources for your virtual machine, your virtual
+machine's workload is scheduled on CPUs that will not be used by other
+processes. By using dedicated resources, you improve the performance of the
+virtual machine and the accuracy of latency predictions.
+
+To enable dedicated resources, the `cpumanager` label must be `true` for each
```

Comment:
This implies that applying the label enables dedicated resources.
Suggest something along the lines of "Only nodes with the `cpumanager=true` label can schedule virtual machines with dedicated resources."

Reply:
I trimmed this from the About section as you suggested earlier.

In the prep section, I now have:

"To prepare to enable dedicated resources, the `cpumanager` label must be `true`
for each node on which you want to schedule virtual machines with
dedicated resources."